File: src/main/java/com/ort/fw/ExceptionControllerAdvice.java
package com.ort.fw;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.web.bind.WebDataBinder;
import org.springframework.web.bind.annotation.ControllerAdvice;
import org.springframework.web.bind.annotation.ExceptionHandler;
import org.springframework.web.bind.annotation.InitBinder;
import org.springframework.web.bind.annotation.ModelAttribute;

@ControllerAdvice
public class ExceptionControllerAdvice {

    // ===================================================================================
    //                                                                          Definition
    //                                                                          ==========
    private static final Logger logger = LoggerFactory.getLogger(ExceptionControllerAdvice.class);

    // ===================================================================================
    //                                                                           Attribute
    //                                                                           =========
    @InitBinder
    public void initBinder(WebDataBinder binder) {
    }

    // ===================================================================================
    //                                                                             Execute
    //                                                                             =======
    @ExceptionHandler(Exception.class)
    public String exception(Exception e) {
        logger.error(e.getStackTrace().toString());
        return "error";
    }

    @ModelAttribute
    public void modelAttribute() {
    }
}

File: backend/src/main/kotlin/com/ort/app/fw/exception/ExceptionControllerAdvice.kt
package com.ort.app.fw.exception

import org.slf4j.LoggerFactory
import org.springframework.web.bind.WebDataBinder
import org.springframework.web.bind.annotation.ExceptionHandler
import org.springframework.web.bind.annotation.InitBinder
import org.springframework.web.bind.annotation.ModelAttribute

class ExceptionControllerAdvice {

    private val logger = LoggerFactory.getLogger(ExceptionControllerAdvice::class.java)

    @InitBinder
    fun initBinder(binder: WebDataBinder) {
    }

    @ExceptionHandler(Exception::class)
    fun exception(e: Exception): String {
        logger.error(e.message, e)
        return "error"
    }

    @ModelAttribute
    fun modelAttribute() {
    }
}